packages/grid/x-data-grid/src/components/menu/columnMenu/GridColumnMenuContainer.tsx

```diff
@@ -2,13 +2,19 @@ import clsx from 'clsx';
 import PropTypes from 'prop-types';
 import * as React from 'react';
 import MenuList from '@mui/material/MenuList';
+import { styled } from '@mui/material/styles';
+
 import { isHideMenuKey, isTabKey } from '../../../utils/keyboardUtils';
-import { GridColumnMenuProps } from './GridColumnMenuProps';
+import { GridColumnMenuContainerProps } from './GridColumnMenuProps';
 import { gridClasses } from '../../../constants/gridClasses';
 
-const GridColumnMenuContainer = React.forwardRef<HTMLUListElement, GridColumnMenuProps>(
-  function GridColumnMenuContainer(props: GridColumnMenuProps, ref) {
-    const { hideMenu, currentColumn, open, id, labelledby, className, children, ...other } = props;
+const StyledMenuList = styled(MenuList)(() => ({
+  minWidth: 248,
```

Comment:
I wonder if it's not a bit too much width on mobile. The column menu feels quite big.